src/components/atoms/date-picker/date-picker.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from "react";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+
+function datePicker() {
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const [selectedDate, setSelectedDate] = useState(null);
```

Comment:
Kan ta in selectedDate och setSelectedDate som props istället för att hålla ett state inuti komponenten